package/lib/compileFunctions.js

```diff
@@ -62,6 +62,18 @@ module.exports = {
         });
       }
 
+      if (funcObject.egress) {
+        _.assign(funcTemplate.properties, {
```

Comment:
Let's use `Object.assign`

Reply:
Just following existing established formatting.
I will come back to these change requests in a few weeks, we will just have to work with our fork for now.